config/default.py

```diff
@@ -9,3 +9,5 @@
 LOG_LEVEL = 'INFO'
 # AWS
 REGION = 'us-west-1'
+# JWT
+EXPIRES_IN_DAYS = 30
```

Comment:
how long should we set this? what factors are we taking into consideration when setting it? 30 seems long. maybe we ask TRON?